int16[] data
---
int64 errorCode
